lib/logstash/inputs/twitter/patches.rb

```diff
@@ -10,54 +8,12 @@ class TwitterPatches
       def self.patch
         verify_version
         patch_json
-        patch_connection_stream
-        patch_request
       end
 
       private
 
       def self.verify_version
-        raise("Incompatible Twitter gem version and the LogStash::Json.load") unless ::Twitter::Version.to_s == "5.15.0"
-      end
-
-      def self.patch_connection_stream
-        ::Twitter::Streaming::Connection.class_eval do
```

Comment:
Would you mind linking to an upstream issue/PR that indicates _why_ these monkey-patch is obsolete with twitter gem v6.2.0?

Reply:
I did a visual verification, first to identify the nature of the patch to start with by comparing with the original code in v5.15.0 and then looking at the code in 6.2.0 and concluded that these 2 patches were not required anymore. Also all tests are passing but TBH I don't know if there are specific tests for these 2 patches purposes.